src/OpenCV_Mat.jl

```diff
@@ -195,25 +195,25 @@ function pixset(img, row::Int, col::Int, value)
   # RGB images (value::  std::vector<double>)
   elseif (cvtypeval(img) == CV_8SC3)
     cppvec = tostdvec(value)
```

Comment:
> Not sure still whether the conversion to std::vec should be done in this function

Yes, I think whether the conversion is made later on or before shouldn't make difference. But unexpectedly I found that the previous implementation led to the errors as mentioned above. Shall I add the details to reproduce the error ?